### ISSUE TITLE
404 not found.

### ISSUE THREAD

**Issue by oreanroy** [UdacityFrontEndScholarship/udabuddy#115]
Design a 404 not found page and through .htacces If unable to implement .htacces ping me on slack.

**Comment by vattytrivedi** [comment]
Can you suggest what kind of design you wish for?
Also some basic on the template if any.
@oreanroy

**Comment by oreanroy** [comment]
you are free to explore your creativity

**Comment by vattytrivedi** [comment]
Here are a few samples of error pages:

- https://codepen.io/VattyTrivedi/pen/VxroRO
- https://codepen.io/VattyTrivedi/pen/LmOwaq
- https://codepen.io/VattyTrivedi/pen/JvOgze
- https://codepen.io/VattyTrivedi/pen/RyyEOM
- https://codepen.io/VattyTrivedi/pen/pVVqmz

**Comment by vattytrivedi** [comment]
@oreanroy  Can you check at the links?

**Comment by oreanroy** [comment]
i like the second and third one

**Comment by vattytrivedi** [comment]
so @oreanroy any suggestions or idea to merge them.
How should I make a PR for this?

**Comment by praveen-me** [comment]
Yeah. @oreanroy.Where can I add my code? Because in the project where I can add just like index.html

**Comment by oreanroy** [comment]
@vattytrivedi first implement .htacess in node and then upload a deployment and finally make pr.

**Comment by oreanroy** [comment]
@praveen-me yes, read this https://stackoverflow.com/questions/6528876/how-to-redirect-404-errors-to-a-page-in-expressjs?utm_medium=organic&utm_source=google_rich_qa&utm_campaign=google_rich_qa

**Comment by littlestar642** [comment]
@oreanroy Any ETA on this?
